README.md

```diff
@@ -81,34 +88,51 @@ We recommend using [VSCode](https://code.visualstudio.com/) with the following e
 
 ## Deployments
 
-Frontend: After running `npm run build`, push to your `main` branch to deploy the app. Heroku uses the contents of the `dist` folder for deployment.
-Backend: Pushing to the `main` branch will automatically trigger a new Heroku build.
+Frontend: After running `npm run build`, push to your `main` branch to deploy the app.
```

Comment:
You don't need to run `npm run build` to deploy. Amplify takes care of this for you.